content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -161,7 +161,7 @@ The built-in signer is part of [`kube-controller-manager`](/docs/reference/comma
 
 To activate the built-in signer, you must pass the `--cluster-signing-cert-file` and `--cluster-signing-key-file` flags.
 
-If you're creating a new cluster, you can use a kubeadm [configuration file](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2):
+If you're creating a new cluster, you can use a kubeadm [configuration file](/docs/reference/config-api/kubeadm-config.v1beta2/):
```

Comment:
Aside: the reader might be none the wiser as to where to put this YAML on their filesystem to have it used. Or maybe they pass it on stdin?

We could clarify that (could be a separate PR).

Reply:
the linked page already includes details (under the "Basics" section) that a file can be passed to kubeadm with "--config".